### PR TITLE
added examples for editing privileges

### PR DESCRIPTION
**Pull request by thefactremains**
editing roles and warning to do so at your own risk

### CODE REVIEW

File: source/deployment/admin-roles.rst [hunk 1]
```diff
@@ -59,15 +59,38 @@ The format of the mmctl command is:
...
 
-``mmctl permissions role assign system_read_only_admin captain-marvel clark-kent``
+**To grant the Read Only Admin role to two users, Bob Smith and Sue Clark:**
+
+``mmctl permissions role assign system_read_only_admin bob-smith sue-clark``
+
+Editing Privileges of Admin Roles (Advanced)
+--------------------------------------------
+
+Each of the admin roles have defined, default privileges as outlined above. However both read and write access to areas of the System Console can be granted or removed from each role by System Admins using the mmctl tool. This can be done either locally or remotely.
```
Comment (justinegeffen) [inline source/deployment/admin-roles.rst]: Does this mean that although there are default settings, those defaults can be modified? For example, can the user manager role be given more access over and above the default access? Can the System Manager have some access removed? If so, is there a table indicating which defaults have been modified?

File: source/deployment/admin-roles.rst [hunk 2]
```diff
@@ -59,15 +59,38 @@ The format of the mmctl command is:
...
+
+**To grant read only access to the authentication section for all users with the User Manager role:**
+
+``mmctl permissions remove system_user_manager sysconsole_read_authentication``
+
+**To remove write access to the authentication section for all users with the User Manager role:**
+
+``mmctl permissions remove system_user_manager sysconsole_write_authentication``
 
 Frequently Asked Questions
```
Comment (justinegeffen) [inline source/deployment/admin-roles.rst:86]: Can we add: "How do I remove a user from a role?"

**Review by mkraft — APPROVED** [review]

File: source/deployment/admin-roles.rst (hunk 1 above)
Comment (thefactremains) [reply source/deployment/admin-roles.rst]: Yes, that is what that means. No, there is not currently a way to see what has been changed from defaults,  but the UI (coming in a future release) will eventually reflect that. We are also planning to add a "reset to defaults" switch.

File: source/deployment/admin-roles.rst (hunk 2 above)
Comment (thefactremains) [reply source/deployment/admin-roles.rst:89]: yes. added under `Assigning Admin Roles`

File: source/deployment/admin-roles.rst (hunk 1 above)
Comment (justinegeffen) [reply source/deployment/admin-roles.rst:73]: Awesome, thank you!

File: source/deployment/admin-roles.rst [hunk 3]
```diff
@@ -59,15 +59,47 @@ The format of the mmctl command is:
...
+``mmctl permissions role assign system_read_only_admin bob-smith sue-clark``
+
+**To remove the System Manager role from a single user called Bob Smith:**
+
+``mmctl permissions role unassign system_manager bob-smith``
+
+Editing Privileges of Admin Roles (Advanced)
+--------------------------------------------
+
+Each of the admin roles have defined, default privileges as outlined above. However both read and write access to areas of the System Console can be granted or removed from each role by System Admins using the mmctl tool. This can be done either locally or remotely.
```
Comment (justinegeffen) [inline source/deployment/admin-roles.rst:73]:
```suggestion
Each of the admin roles have defined, default privileges as outlined above. 

System Admins can grant read and write access to other areas of the System Console, as well as remove read write access (including default access), for each role. This is done via the mmctl tool, either locally or remotely.
```

File: source/deployment/admin-roles.rst [hunk 4]
```diff
@@ -45,7 +45,7 @@ Each role has a set of default permissions, and when a user is assigned a role,
 Assigning Admin Roles
 ---------------------
 
-System Admins can assign roles and modify privileges using the mmctl tool. This can be done either locally or remotely.
+System Admins can assign roles using the mmctl tool. This can be done either locally or remotely.
```
Comment (justinegeffen) [inline source/deployment/admin-roles.rst:48]:
```suggestion
System Admins can assign roles using the mmctl tool, either locally or remotely.
```

File: source/deployment/admin-roles.rst [hunk 5]
```diff
@@ -59,15 +59,47 @@ The format of the mmctl command is:
...
+Editing Privileges of Admin Roles (Advanced)
+--------------------------------------------
+
+Each of the admin roles have defined, default privileges as outlined above. However both read and write access to areas of the System Console can be granted or removed from each role by System Admins using the mmctl tool. This can be done either locally or remotely.
+
+The format of the mmctl command is:
+
+``mmctl permissions add [role_name] [permission...]``
+
+**To grant write access to the authentication section for all users with the User Manager role:**
```
Comment (justinegeffen) [inline source/deployment/admin-roles.rst:79]:
```suggestion
**To grant write access to the Authentication area of the System Console for all users with the User Manager role:**
```

File: source/deployment/admin-roles.rst [hunk 6]
```diff
@@ -59,15 +59,47 @@ The format of the mmctl command is:
...
+
+``mmctl permissions add system_user_manager sysconsole_write_authentication``
+
+**To grant read only access to the authentication section for all users with the User Manager role:**
```
Comment (justinegeffen) [inline source/deployment/admin-roles.rst:83]:
```suggestion
**To grant read only access to the Authentication area of the System Console for all users with the User Manager role:**
```

**Comment by justinegeffen** [comment]
A few suggestions but otherwise looking good! :)

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**